### ISSUE TITLE
Tutorials: Continuing list numbering

### ISSUE THREAD

**Issue by jonnilundy** [wedeploy/wedeploy.com#105]
<img width="371" alt="screen shot 2017-03-30 at 2 29 02 pm" src="https://cloud.githubusercontent.com/assets/23219848/24527052/709b2d00-1555-11e7-8a5c-52529d4e2330.png">
<img width="298" alt="screen shot 2017-03-30 at 2 29 11 pm" src="https://cloud.githubusercontent.com/assets/23219848/24527053/709ef408-1555-11e7-913e-5dcf0e6659e6.png">

**Comment by zenorocha** [comment]
Related to https://github.com/electricjs/electric/issues/36

**Comment by ygorcosta** [comment]
Hey @jonnilundy,
To not stay blocked while that issue is not done, you can use the same thing that I used in docs (I had to use html instead md)
For the above code, you can do this.
```
<ol>
<li>Add your fork's</li>
<ul>
<li><strong>owner</strong> (you)</li>
<li><strong>repo</strong> (oscon-game)</li>
<li><strong>branch</strong> (master)</li>
</ul>
<li>Click <em>Connect Repository</em></li>
</ol>
```

**Comment by jonnilundy** [comment]
Thanks @ygorcosta!

**Comment by jonnilundy** [comment]
@ygorcosta I just used that code and it didn't seem to change anything. Is it something particular to the docs?

**Comment by ygorcosta** [comment]
I'm taking a look on that right now

**Comment by ygorcosta** [comment]
@jonnilundy I fixed that on marble v2.9.6
